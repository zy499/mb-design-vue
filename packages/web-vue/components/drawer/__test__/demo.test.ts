import demoTest from '../../../scripts/demo-test';

demoTest('drawer');
