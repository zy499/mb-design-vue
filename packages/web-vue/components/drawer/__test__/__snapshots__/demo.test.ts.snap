// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<drawer> demo: render [async] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Drawer
</button>
<!---->"
`;

exports[`<drawer> demo: render [basic] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Drawer
</button>
<!---->"
`;

exports[`<drawer> demo: render [nested] correctly 1`] = `
"<button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Drawer
</button>
<!---->
<!---->"
`;

exports[`<drawer> demo: render [popup-container] correctly 1`] = `
"<div>
  <div id=\\"parentNode\\" style=\\"width: 100%; height: 300px; position: relative; overflow: hidden; line-height: 300px; text-align: center;\\"><button class=\\"mb-btn mb-btn-primary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
      <!--v-if-->Open Drawer
    </button></div>
</div>
<!---->"
`;

exports[`<drawer> demo: render [position] correctly 1`] = `
"<span class=\\"mb-radio-group mb-radio-group-size-undefined mb-radio-group-direction-horizontal\\"><label class=\\"mb-radio\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"top\\"><span class=\\"mb-icon-hover mb-radio-icon-hover\\"><span class=\\"mb-radio-icon\\"></span></span><span class=\\"mb-radio-label\\">Top</span></label><label class=\\"mb-radio mb-radio-checked\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"right\\"><span class=\\"mb-icon-hover mb-icon-hover-disabled mb-radio-icon-hover\\"><span class=\\"mb-radio-icon\\"></span></span><span class=\\"mb-radio-label\\">Right</span></label><label class=\\"mb-radio\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"bottom\\"><span class=\\"mb-icon-hover mb-radio-icon-hover\\"><span class=\\"mb-radio-icon\\"></span></span><span class=\\"mb-radio-label\\">Bottom</span></label><label class=\\"mb-radio\\"><input type=\\"radio\\" class=\\"mb-radio-target\\" value=\\"left\\"><span class=\\"mb-icon-hover mb-radio-icon-hover\\"><span class=\\"mb-radio-icon\\"></span></span><span class=\\"mb-radio-label\\">Left</span></label></span><button class=\\"mb-btn mb-btn-secondary mb-btn-shape-square mb-btn-size-medium mb-btn-status-normal\\" type=\\"button\\">
  <!--v-if-->Open Drawer
</button>
<!---->"
`;
